apps/pigment-css-next-app/package.json

```diff
@@ -10,18 +10,18 @@
   },
   "dependencies": {
     "@pigment-css/react": "workspace:^",
-    "@mui/material": "^5.16.6",
-    "@mui/material-nextjs": "^5.16.6",
+    "@mui/material": "^6.1.6",
+    "@mui/material-nextjs": "^6.1.6",
     "local-ui-lib": "workspace:^",
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
-    "next": "latest"
+    "next": "15.0.2"
   },
   "devDependencies": {
     "@pigment-css/nextjs-plugin": "workspace:^",
-    "@types/node": "^20.5.7",
-    "@types/react": "^18.3.3",
-    "@types/react-dom": "^18.2.19",
+    "@types/node": "^18.19.63",
```

Comment:
This's a downgrade. Is this correct?

Reply:
It is. v20 never actually was used since at the root, in `resolutions`, it was still v18. I have kept it at 18 since we are currently using node v18 to build in CI.